Comp2024/src/main/java/frc/robot/subsystems/HID.java

```diff
@@ -106,11 +120,31 @@ public void printFaults( )
    * @param intensity
    *          requested rumble strength
    */
-  private void setHIDRumble(boolean driverRumble, boolean operatorRumble, double intensity)
+  private void setHIDRumbleDriver(boolean driverRumble, double intensity)
   {
-    DataLogManager.log(
-        String.format("%s: Rumble driver: %s operator: %s intensity: %.1f", getName( ), driverRumble, operatorRumble, intensity));
+    DataLogManager.log(String.format("%s: Rumble driver: %s intensity: %.1f", getName( ), driverRumble, intensity));
+
+    m_driverRumbleOn = driverRumble;
+
+    if (m_driverRumbleOn)
+    {
+      m_timerDriver.restart( );
+    }
+
     m_driver.setRumble(RumbleType.kBothRumble, (driverRumble) ? intensity : 0.0);
+  }
+
+  private void setHIDRumbleOperator(boolean operatorRumble, double intensity)
+  {
+    DataLogManager.log(String.format("%s operator: %s intensity: %.1f", getName( ), operatorRumble, intensity));

```

Comment:
Maybe add in the "Rumble" like you did in the driver log message to keep them the same format. See your log line above.

Reply:
fixed